myuw/data/resource_link_import.csv

```diff
@@ -1,154 +1,168 @@
+Teaching,Tools,all,https://canvas.uw.edu,Canvas LMS,,,,,,,no
+Teaching,Tools,all,https://catalyst.uw.edu/,Catalyst Web Tools,,,,,,,no
+Teaching,Tools,all,https://uw.hosted.panopto.com,Panopto Lecture Capture,,,,,,,no
+Teaching,Tools,all,https://itconnect.uw.edu/learn/tools/polleverywhere/,Poll Everywhere,,,,,,,no
+Teaching,Tools,all,https://coda.uw.edu,Course Stats,,,,,,,no
+Teaching,Tools,all,https://washington.zoom.us,Zoom,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/,UW Research Website,,,,,,,no
+Research,Research Administration,all,,,https://www.washington.edu/research/new-to-uw-research/,New to UW Research,https://www.uwb.edu/research/support,New to UW Research,https://www.tacoma.uw.edu/or/conducting-research-uw-tacoma,New to UW Research,no
+Research,Research Administration,all,https://www.washington.edu/research/myresearch-lifecycle/,MyResearch Project Lifecycle,,,,,,,no
+Research,Research Administration,all,https://myresearch.washington.edu/,MyResearch Portal,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/tools/sage/,System to Administer Grants Electronically (SAGE),,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/tools/,Research Administration Tools,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/resources/funding-opportunities/,Funding Opportunities,,,,,,,no
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://www.washington.edu/research/research-centers/,Research Centers and Institutes,,,,,,,no
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://www.washington.edu/research/shared-research-facilities-resources/,Shared Research Facilities and Resources,,,,,,,no
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://escience.washington.edu/,eScience Institute,,,,,,,no
+Research,Research Data and Publishing,all,https://www.lib.washington.edu/services/faculty,UW Libraries: Getting started with your research,,,,,,,no
+Research,Research Data and Publishing,all,https://www.lib.washington.edu/dataservices/tools/services,Tools for Data-Related Research,,,,,,,no
+Research,Research Data and Publishing,all,https://guides.lib.uw.edu/research/dmg,Research Data Services,,,,,,,no
+Research,Research Data and Publishing,all,https://hsl.uw.edu/trail/,Translational Research and Information Lab (TRAIL),,,,,,,no
+Research,Research Data and Publishing,all,,,https://www.lib.washington.edu/openscholarship,"Open Scholarship Commons: Tools, Services, Consultations",https://guides.lib.uw.edu/bothell/digitalscholarship,Digital Scholarship,https://guides.lib.uw.edu/tacdigitalscholarship,Digital Scholarship,no
+Research,Research Data and Publishing,all,https://guides.lib.uw.edu/research/spoa,Support for Publishing,,,,,,,no

```

Comment:
Would "Scholarly Publishing and Open Access" be a more fitting title here?

Reply:
Yes I agree! Updated to "Scholarly Publishing and Open Access" ! Thanks :)